server/protocols/jmap-rfc-8621/src/main/scala/org/apache/james/jmap/method/EmailGetMethod.scala

```diff
@@ -112,11 +113,17 @@ class EmailGetMethod @Inject() (serializer: Serializer,
     request.properties match {
       case None => Right(Email.defaultProperties)
       case Some(properties) =>
-        val invalidProperties = properties -- Email.allowedProperties
-        if (invalidProperties.isEmpty()) {
+        val mayContainsSpecificHeaders = properties -- Email.allowedProperties
+        if (mayContainsSpecificHeaders.isEmpty()) {
           Right(properties ++ Email.idProperty)
         } else {
-          Left(new IllegalArgumentException(s"The following properties [${invalidProperties.format()}] do not exist."))
+          val invalidProperties = mayContainsSpecificHeaders.value
+            .filter(p => !p.value.startsWith(SPECIFIC_HEADER_PREFIX) || p.substring(SPECIFIC_HEADER_PREFIX.length).contains(":"))
```

Comment:
Rely on the previously introduced : 

```
object SpecificHeaderRequest {
   def from(property: Property): Option[SpecificHeaderRequest] = ???
}
```

to avoid leaking techy hacky stuff here.